docs/data-prep/ingest-data-fs.md

```diff
@@ -183,16 +183,17 @@ Tech Preview
 ```
 ```{admonition} Limitation
 Do not use SQL reserved words as entity names. See more details in [Keywords and Reserved Words](https://dev.mysql.com/doc/refman/8.0/en/keywords.html).
+Current supported version of SQLAlchemy see extra `pip install mlrun[sqlalchemy]`. See more details about [Dialects](https://docs.sqlalchemy.org/en/20/dialects/index.html) 
```

Comment:
Do you mean - 
Check the current supported version of SQLAlchemy in [extras-requirements.txt](https://github.com/mlrun/mlrun/blob/development/extras-requirements.txt) and if necessary run `pip install mlrun[sqlalchemy]`  to update.

Reply:
Exactly